### DISCUSSION TITLE
modal didnt open

### DISCUSSION THREAD

**Discussion by hai-shift** [wire-elements/modal#366] [Q&A]
hi
i try to use the pack with livewire 3 and laravel 10
and i put that:

`
        <button onclick="Livewire.dispatch('openModal', { component: 'modal.otp-valid'})">Edit User</button>
`

in livewire dubug i see that update:
`
2 templates were rendered
wire-elements-modal::modal (resources/views/vendor/wire-elements-modal/modal.blade.php)
livewire.modal.otp-valid (resources/views/livewire/modal/otp-valid.blade.php)
`
in my layout i put that:

`
    @livewireScriptConfig
    @livewire('wire-elements-modal')
    </body>
</html>
`

and i get this in the console sometimes:
`checked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received`

but the popup didnt showing in the page

what can i do?

**Comment by lrljoe** [comment]
Looks like a CORS failure.

Test with a browser without any plugins enabled, as they often cause that.
